src/htm/algorithms/TemporalMemory.hpp

```diff
@@ -236,6 +236,14 @@ class TemporalMemory : public Serializable
     activateDendrites(learn, externalPredictiveInputsActive, externalPredictiveInputsWinners);
   }
 
+  /**
+	* Calculate anomaly score depending on the mode given at initialization
+	*
+	* @param activeColumns
+	* currently activeColumns
+	*/
+  static void calculateAnomalyScore_(const SDR &activeColumns);
```

Comment:
you're right it cannot be const. The same way it does not make sense to be `static`, by private, I mean `private:` access/visibility within the hpp file.

Reply:
yes, i corrected it just now